km/km_coredump.h

```diff
@@ -130,7 +130,7 @@ typedef struct km_nt_file {
    Elf64_Off data;          // depends on file type.
    Elf64_Word datalength;   // bytes of pipe contents following filename
    // Followed by file name
-   // Followed by pipe or socketpair contents
+   // Followed by data buffered in a pipe
```

Comment:
My point was more about the comment on line 132 and this one, being confused when looked at together